openapi.yaml

```diff
@@ -359,7 +359,7 @@ paths:
           schema:
             type: integer
 
-        - name: maxHours
+        - name: maxTravelTime
```

Comment:
maybe change unit to minutes to give users more fine grained control?

Reply:
Sure, I can change that. Using floats works too, but that might not be intuitive for everyone.